doc/tc-meetings/2015-01-07.md

```diff
@@ -0,0 +1,156 @@
+    mikeal: issue re release?
+    rod: not yet, a lot of little bugs to get CI working consistently.  No grand plan for what’s in release yet.
+
+
+### Review of last meeting
+
+* sys: Remove after 3 years of deprecation #182 
+* module: force require('process') to return a reference to process #206
+* File copyright policy #216
+* Rename v0.12 to v1.0.0
+* Merge strategy (v0.10 and joyent/node)
+
+### Invite Colin Ihrig to the TC #223 
+
+https://github.com/iojs/io.js/issues/223
+
+Asked for vote
+
+Passed unanimously
+
+Some discussion about TC member addition process
+
+### File copyright policy #216
+
+https://github.com/iojs/io.js/issues/216
+
+Rod: got feedback from NodeSource GC on this, recommendation is to keep the copyright but removal of the license notice is acceptable
+Mikeal & Isaac: removing the copyright should be OK because we have full git record of who made & edited the file, it’s a PITA
+
+Asked for vote on removing copyright and license from the top of each source file, moving it a single top level file. **Nobody objected**.
+
+Agreed to table further discussion on modifying the license and copyright.
+
+isaacs will submit PR to remove lic blocks from files.
+
+### Doc: clarified & split up contribution docs #233 
+
+https://github.com/iojs/io.js/pull/233
+
+Rod: discussed changes introduced in #233, this is in TC because there are some minor modifications to the governance structure. Recommended that we deal with further changes to governance as a separate discussion.
+
+Fedor: remove caine block from CONTRIBUTING.md
+
+Mikeal: take PR as is, move the rest to further discussion & PR
+
+No objections, agreed to move changes to governance and other adjustments such as moving docs into the docs dir to separate pull requests.
+
+Isaacs and Chris will review and +1 the PR.
+
+
+### Governance: Add new Collaborators #234 
+
+https://github.com/iojs/io.js/issues/234
+
+Rod discussed the generation of the list of proposed collaborators. 
+
+Long discussion:
+Isaacs, Chris, Bert suggest throttling new contrib additions to 5/week, bump up to 10/week if it goes well, etc.
+Ben 50/50 on “add everyone now” or “throttle new additions”
+Bert to review list and see if there’s any contributors that seem we should not add.
+Other TC members can review list this week.
+Two issues to vote on at next TC meeting
+Issue 1:
+A) Add everyone
+B) Add everyone except some that are not added
+Issue 2:
+A) Create a queue of people to add, throttle per week
+B) Add immediately
+
+### deps: upgrade v8 to 3.31.74.1 #243 
+
+https://github.com/iojs/io.js/pull/243
+
+Ben: clean compile, apparently a clean upgrade. It does break postmortem support so Ben removed it completely. Asked who was using it.
```

Comment:
could you list for me here what's in it and I'll amend? I didn't catch it at the time and it's probably going to be easier for you to list than me to find the spot in the video for this